StabilityPackage.wl

```
@@ -400,7 +400,7 @@ Return[out] (*Close reap over r*)
 
 
 getMoments[file_,r_,species_]:= Module[{data,datasr,moments},
-data=ImportData[file];
+data=ImportData[file]//Quiet;

```

The better solution would be to have separate ImportMomentData and ImportBinData functions. Suppressing errors is usually sweeping something under the rug that can come back to bite.

---

This is essentially what the getMoments function does, without having to make new (and redundant) associations.  The quiet isn't a supression of error (ok, technically it is an "error") it just quiets this notification that there aren't mid points for the theta, phi, and energy grids.  I'll remove the quiet just as an assurance there aren't any other errors.